get_id.js

```diff
@@ -107,55 +117,58 @@ function put_value_values(name, placeholder) {
 function parse_disperse_filler (resp) {
     let massivofplaceholders = ["marka", "manufacturer", "chemical_nature", "chemical_nature_iso", "purity_of_disperse_filler", "purity_of_disperse_filler_iso", "dispersity", "dispersity_iso", "moisture_content", "moisture_content_iso", "density", "density_iso"]
     massivofvalue = undefined

```

Comment:
rm. not necessary store 'undefined' into value 
Use:
do_foreach (resp, massivofplaceholders);